Gems/PhysX/Code/Source/RigidBody.cpp

```diff
@@ -83,6 +83,8 @@ namespace PhysX
             {
                 SetUserData(configuration.m_customUserData);
             }
+
+            SetAxisLockFlags(configuration.m_linearAxisLock, configuration.m_angularAxisLock);
```

Comment:
Given that the values (0/1) for the locking properties are coming from the `RigidBodyConfiguration` object, would it not make more sense to move this to `PxActorFactories::CreatePxRigidBody`? That's where I put the logic for my quick (hacky) fix. I also made the code that upgrades older serialized versions add appropriate defaults (which may not actually be required) to make it safe to move this to `PxActorFactories::CreatePxRigidBody`.